cluster-autoscaler/cloudprovider/gke/gke_cloud_provider.go

```diff
@@ -225,33 +226,38 @@ type gkeMig struct {
 }
 
 // GceRef returns Mig's GceRef
-func (mig *gkeMig) GceRef() gce.GceRef {
+func (mig *GkeMig) GceRef() gce.GceRef {
 	return mig.gceRef
 }
 
+// Name returns the name of the Mig.
+func (mig *GkeMig) Name() string {
```

Comment:
nit: is this really necessary? This is equivalent to GceRef().Name

Reply:
Ok I'm oficially blind, sorry.